src/Testcontainers.PostgreSql/PostgreSqlBuilder.cs

```diff
@@ -123,18 +123,10 @@ protected override PostgreSqlBuilder Merge(PostgreSqlConfiguration oldValue, Pos
     /// <inheritdoc cref="IWaitUntil" />
     private sealed class WaitUntil : IWaitUntil
     {
-        private static readonly string[] LineEndings = ["\r\n", "\n"];
-
         /// <inheritdoc />
-        public async Task<bool> UntilAsync(IContainer container)
+        public Task<bool> UntilAsync(IContainer container)
         {
-            var (stdout, stderr) = await container.GetLogsAsync(timestampsEnabled: false)
-                .ConfigureAwait(false);
-
-            return 2.Equals(Array.Empty<string>()
-                .Concat(stdout.Split(LineEndings, StringSplitOptions.RemoveEmptyEntries))
-                .Concat(stderr.Split(LineEndings, StringSplitOptions.RemoveEmptyEntries))
-                .Count(line => line.Contains("database system is ready to accept connections")));
+            return ((PostgreSqlContainer)container).IsReadyAsync();
         }
     }
```

Comment:
Can we please follow the pattern we are already using in other moduls (e.g. MySQL [1](https://github.com/testcontainers/testcontainers-dotnet/blob/f4ee0b2d4cd0d02d11d1e1ecceb90c6d521728e7/src/Testcontainers.MySql/MySqlBuilder.cs#L78-L81), [2](https://github.com/testcontainers/testcontainers-dotnet/blob/f4ee0b2d4cd0d02d11d1e1ecceb90c6d521728e7/src/Testcontainers.MySql/MySqlBuilder.cs#L128-L148))? I prefer a consistent approach; it is easier to navigate and refactor the code.

Reply:
Absolutely, I just addressed this in bf08a0ec9865fd1bdf23d34c5d20bca85cca9e19.

Reply:
I added `.ConfigureAwait(false)` and force-pushed so that's now commit cf0fdf109940b6bd6df0c4467849da4284b2409c.